torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -3701,6 +3701,7 @@ c10::intrusive_ptr<Work> ProcessGroupNCCL::allreduce_sparse(
 #endif
 }
 
+template <bool IsBarrier>
```

Comment:
nit: start with lowerCase variable name (isBarrier) to be consistent with other variable definition?